dockstore-webservice/src/main/java/io/dockstore/webservice/core/metrics/MetricsByStatus.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright 2024 OICR and UCSC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *           http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package io.dockstore.webservice.core.metrics;
+
+import io.swagger.annotations.ApiModelProperty;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.media.Schema.RequiredMode;
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.NotNull;
+
+@Entity
+@Table(name = "metrics_by_status")
+@Schema(name = "MetricsByStatus", description = "Aggregated metrics grouped by execution status")
+public class MetricsByStatus {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @ApiModelProperty(value = "Implementation specific ID for the metrics in this webservice")
+    @Schema(description = "Implementation specific ID for the metrics in this webservice")
+    private long id;
+
+    @Column(nullable = false)
+    @NotNull
+    @Schema(description = "The number of executions for the status", requiredMode = RequiredMode.REQUIRED)
+    private int executionStatusCount;
+
+    @Valid
+    @OneToOne(cascade = CascadeType.ALL, orphanRemoval = true)
+    @JoinColumn(name = "executiontime", referencedColumnName = "id")
+    @ApiModelProperty(value = "Aggregated execution time metrics in seconds")
+    @Schema(description = "Aggregated execution time metrics in seconds")
```

Comment:
These are actually IDs for the metric rather than the metric itself (similar to https://github.com/dockstore/dockstore/blob/7b3ffed65383593f08cbc749f2762cfcbe7bab61/dockstore-webservice/src/main/java/io/dockstore/webservice/core/BioWorkflow.java#L62 )
It may make sense to describe it as an ID for execution time to avoid duplicating the description which is incomplete anyway since the unit can be overridden.

Reply:
May also make sense to describe the variable like that 
i.e. `executionTimeMetricID` to hint at the purpose of the column when writing sql queries

Reply:
I changed the DB column names to include the word `id`, like `executiontimeid`